horreum-client/src/test/resources/testcontainers/horreum.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```

Comment:
We've discussed that it would be better not to duplicate these scripts.

Reply:
I have not finished cleaning this up yet. They are not strictly duplicated so am working on unifying their behaviour atm

Reply:
Sorry, no rush. I just thought it's up for the review when you've pushed the changes.
Btw. when you're done please squash the changes and rebase.